lib/middlewares.js

```diff
@@ -3,21 +3,58 @@ const shortid = require('shortid');
 
 const { namespace, setRequestId } = require('./namespace');
 
-const expressMiddleware = opts => (req, res, next) => {
-  const { method, params, query, body } = req;
-  const url = req.originalUrl || req.url;
-  const { loggerFn } = opts || {};
+const secure = '[SECURE]';
 
-  loggerFn(`Started ${url} ${method} with params:`, params || {}, 'query:', query || {}, 'body:', body || {});
-  const begin = Date.now();
-  const onFinish = namespace.bind((error, response) => {
-    const end = Date.now();
-    const responseTime = error ? '-' : end - begin;
-    const status = response.statusCode;
-    loggerFn(`Ended ${method} ${url} with status: ${status} in ${responseTime} ms`);
-  });
-  onFinished(res, onFinish);
-  next();
+const getSecureBody = ({ params, url, method, body, placeholder }) => {
+  try {
+    if (params === true) {
```

Comment:
can we clarify why do we need this to be exactly true?

Reply:
Because this could be an object or boolean (see [this](https://github.com/Wolox/express-wolox-logger/tree/fix-secrets-v1#obfuscatebody-objectboolean)), if we check truthy both passes condition